pkg/gadgets/run/types/metadata.go

```diff
@@ -210,6 +227,20 @@ func (m *GadgetMetadata) validateStructs(spec *ebpf.CollectionSpec) error {
 	return result
 }
 
+func (m *GadgetMetadata) validateParams(spec *ebpf.CollectionSpec) error {
+	var result error
+	for varName := range m.EBPFParams {
+		err := checkParamVar(spec, varName)
+		if err != nil {
```

Comment:
nit: `if err := checkParamVar(spec, varName); err != nil`. 

I see the same can be done in other places. (However it's just a personal preference, if you think it's better as it's ignore this comment)